_includes/header.html

```diff
@@ -6,18 +6,44 @@
 </span>
 <span class="right">
     {% for item in site.menu %}
+    {% if item.children %}
+    <div class="nav dropdown">
+    <div class="dropdown-btn">
+    <button>{{ item.name | capitalize }}</button>
+    </div>
+    <div class="dropdown-content">
+      {% for sub in item.children %}
+      {% assign link = item.name | append: '/' | append: sub %}
+      <a class="nav" href="{{ link | relative_url }}">{{ sub }}</a>
```

Comment:
I've noticed the annoying twitching whenever you hover over the menu elements in the dropdown because of the fancy animation thing whenever you hover over stuff in the navbar. 

I've looked into it and it seems pretty annoying to fix, so to me the easiest solution is to just remove the `nav` class, but if you can manage to fix it then even better